### PR TITLE
Automatic backup for sites

### PR DESCRIPTION
**Pull request by alaa-alsalehi**
As a person who manages a multitenancy system, I want to be able to backup sites. So we added an option for every site to repeat the backup operation based on time with three options: daily, weakly, and monthly.

### CODE REVIEW

**Comment by alaa-alsalehi** [comment]
@gavindsouza @mymi14s check this PR please

File: bench_manager/bench_manager/doctype/bench_settings/bench_settings.py [hunk 1]
```diff
@@ -383,3 +383,31 @@ def run_command(commands, doctype, key, cwd="..", docname=" ", after_command=Non
 			docname=docname,
 			commands=commands,
 		)
+
+def backup_sites_with_daily_option():
+    site_list = frappe.get_list("Site",filters={"frequancy":"Daily","auto_backup":1})
```
Comment (mymi14s) [inline bench_manager/bench_manager/doctype/bench_settings/bench_settings.py]: could you please correct the spelling frequancy => frequency

Comment (mymi14s) [reply bench_manager/bench_manager/doctype/bench_settings/bench_settings.py]: @ServeMe please see the above.

Comment (alaa-alsalehi) [reply bench_manager/bench_manager/doctype/bench_settings/bench_settings.py]: sure @bahaaabed please fix it

Comment (ServeMe) [reply bench_manager/bench_manager/doctype/bench_settings/bench_settings.py:388]: @mymi14s Does enqueue the backup the suite option for the situation? or do we need to do any other move?
suppose we have 100 sites does that freeze up the whole server?

Comment (alaa-alsalehi) [reply bench_manager/bench_manager/doctype/bench_settings/bench_settings.py]: @mymi14s typo mistake fixed

**Comment by alaa-alsalehi** [comment]
@mymi14s Does enqueue the backup the suite option for the situation? or do we need to do any other move?
suppose we have 100 sites does that freeze up the whole server?

**Comment by mymi14s** [comment]
It could freeze the server if limited resources, to improve performance, increase background workers

**Comment by alaa-alsalehi** [comment]
OK looks good I think we can approve this for the current time and in future we think how to improve it

**Review by mymi14s — APPROVED** [review]

**Pull request merged**